app/views/prosecution_cases/show.html.haml

```diff
@@ -15,7 +15,10 @@
       - cse.defendants.each do |defendant|
         %tr.govuk-table__row
           %td.govuk-table__cell
-            = link_to defendant.name, edit_defendant_path(defendant.arrest_summons_number || defendant.national_insurance_number)
+            - if defendant.linked?
```

Comment:
consider using a view helper/decorator of some kind

Reply:
👍 created defendant_link_helper.rb